RHIDP-5409 Consistent titles and subtitles (#832)
redhat-developer/red-hat-developers-documentation-rhdh: titles/configuring/master.adoc

include::artifacts/attributes.adoc[]
:title: Configuring {product}

> :subtitle: Configuring {product} for production to work in your IT ecosystem by adding custom config maps and secrets

Author: ```suggestion
:subtitle: Add custom config maps and secrets to configure your {product} instance to work in your IT ecosystem.
```
